src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,262 @@
+package kitchenpos.application;
+
+import kitchenpos.MemoryPurgomalumClient;
+import kitchenpos.domain.*;
+import kitchenpos.infra.PurgomalumClient;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.math.BigDecimal;
+import java.util.*;
+import java.util.stream.Stream;
+
+import static kitchenpos.KitchenposFixture.*;
+import static kitchenpos.fixture.MenuFixture.*;
+import static kitchenpos.fixture.MenuGroupFixture.정상_메뉴_그룹;
+import static kitchenpos.fixture.ProductFixture.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class MenuServiceTest {
+
+  private static final Long NEGATIVE_NUM = -1L;
+
+  @Mock
+  MenuRepository menuRepository;
+  @Mock
+  MenuGroupRepository menuGroupRepository;
+  @Mock
+  ProductRepository productRepository;
+  @Mock
+  MemoryPurgomalumClient purgomalumClient;
+
+  @InjectMocks
+  private MenuService menuService;
+
+  private static Stream<BigDecimal> menuPriceNullAndMinus() {
+    return Stream.of(
+      BigDecimal.valueOf(NEGATIVE_NUM),
+      null
+    );
+  }
+
+  @Test
+  @DisplayName("가게 점주는 메뉴를 추가 할 수 있습니다.")
+  void addMenu() {
+    //given
+    Menu request = 정상_메뉴_가격_만원();
+    MenuGroup menuGroup = 정상_메뉴_그룹();
+    Product product = 상품_가격_만원();
+    List<Product> productList = Collections.singletonList(product);
+
+    when(menuGroupRepository.findById(any())).thenReturn(Optional.of(menuGroup));
+    when(productRepository.findAllByIdIn(any())).thenReturn(productList);
+    when(productRepository.findById(any())).thenReturn(Optional.of(product));
+
+    //then
+    assertDoesNotThrow(() -> {
+      Menu menu = menuService.create(request);
+      verify(menuRepository).save(any(Menu.class));
+    });
```

Comment:
assertDoesNotThrow 와 분리하면 좋을것 같아요 ! 
```suggestion
    assertDoesNotThrow(() -> {
      Menu menu = menuService.create(request);
    });
    verify(menuRepository).save(any(Menu.class));
```